definitions/features/hammer.rb

```diff
@@ -70,6 +70,10 @@ def run(args)
     execute("#{command_base} #{args}")
   end
 
+  def reload_cache
+    execute("#{command_base} --reload-cache")
```

Comment:
@kgaikwad as you mentioned mostly this should not cause any issue when hammer is unable to connect to server as hammer feature takes care of it, on my system where hammer is failing with warnings it takes 1 min to complete the command; so maybe we can add the spinner but its not mandatory.

Reply:
> @kgaikwad as you mentioned mostly this should not cause any issue when hammer is unable to connect to server as hammer feature takes care of it

Tested this scenario. 
Instead of calling `execute` method directly, either I have to call `run` method  with args `--reload-cache` or I need to give `setup_admin_access` call before `execute` method call. This way it will handle server connection part.
Thank you for pointing this out :+1: 

> on my system where hammer is failing with warnings it takes 1 min to complete the command; so maybe we can add the spinner but its not mandatory.

Yes, I can add spinner.

Reply:
Done!